Chocolatey-AU.build.ps1

```diff
@@ -0,0 +1,375 @@
+            throw "You are not an administrator. We cannot use Chocolatey to install gitversion.portable."
+        }
+
+        choco install gitversion.portable -y --no-progress
+    }
+}
+
+# Synopsis: ensure PowerShellGet has the NuGet provider installed
+task BootstrapPSGet {
+    if (-not (Get-PackageProvider NuGet -ErrorAction Ignore)) {
+        Write-Host "Installing NuGet package provider"
+        Install-PackageProvider NuGet -MinimumVersion 2.8.5.201 -ForceBootstrap -Force
+    }
+
+    Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+
+    if (-not (Get-InstalledModule PowerShellGet -MinimumVersion 2.0 -MaximumVersion 2.99 -ErrorAction Ignore)) {
+        Install-Module PowerShellGet -MaximumVersion 2.99 -Force -AllowClobber -Scope CurrentUser
+        Remove-Module PowerShellGet -Force
+        Import-Module PowerShellGet -MinimumVersion 2.0 -Force
+        Import-PackageProvider -Name PowerShellGet -MinimumVersion 2.0 -Force
+    }
+}
+
+# Synopsis: ensure Pester is installed
+task InstallPester BootstrapPSGet, {
+    if (-not (Get-InstalledModule Pester -MaximumVersion 4.99 -ErrorAction SilentlyContinue)) {
+        Write-Host "Installing Pester"
+        Install-Module Pester -MaximumVersion 4.99 -SkipPublisherCheck -Force -Scope CurrentUser -ErrorAction Stop -Verbose:$false
+    }
+}
+
+# Synopsis: ensure PSScriptAnalyzer is installed
+task InstallScriptAnalyzer BootstrapPSGet, {
+    if (-not (Get-InstalledModule PSScriptAnalyzer -MinimumVersion 1.20 -ErrorAction SilentlyContinue)) {
+        Write-Host "Installing PSSA"
+        Install-Module PSScriptAnalyzer -Scope CurrentUser -Force -MinimumVersion 1.20 -ErrorAction Stop -Verbose:$false
+    }
+}
+
+# Synopsis: cleanup build artifacts
+task Clean {
+    remove $OutputDirectory
+    New-Item -Path $OutputDirectory -ItemType Directory | Out-Null
+}
+
+# Synopsis: run PSScriptAnalyzer on project files
+task ScriptAnalyzer InstallScriptAnalyzer, {
+    $results = Invoke-ScriptAnalyzer -Path $script:SourceFolder -Recurse -Settings "$PSScriptRoot/PSScriptAnalyzerSettings.psd1"
+    if ($results) {
+        Write-Warning "$($results.Count) PSSA rule violations found."
+        $results
+        # Chocolatey-AU currently has lots of PSSA violations. None of them are errors in PSSA.
+        # For now, the build will not fail on PSSA unless asked.
+        if ($ThrowOnPSSAViolation) {
+            throw "PSSA rule violations detected, see above errors for more information"
+        }
+    }
+}
+
+# Synopsis: build the project
+task Build Clean, InstallGitVersion, ScriptAnalyzer, {
+    New-Item $script:ModuleOutputDir -ItemType Directory | Out-Null
+
+    Copy-Item "$script:SourceFolder/*" -Destination $script:ModuleOutputDir -Recurse
+    $manifest = Get-ChildItem "$OutputDirectory/$ModuleName/$ModuleName.psd1"
+
+    $gitversion = gitversion.exe
+    $gitversion | Out-String -Width 120 | Write-Host
+    $versionInfo = $gitversion | ConvertFrom-Json
```

Comment:
This is the problem with the build. You are assuming that GitVersion would only output the JSON itself, but that isn't always true.

To get it working, you need to ignore whatever is outputted to `stderr`, so in theory you an do

```suggestion
    $versionInfo = $gitversion 2>$null | ConvertFrom-Json
```

Reply:
I've added a commit with this in it, will monitor the GHA run.

Reply:
It seems the redirect to null didn't solve it, but the update of `GitVersion.yml` did 😁. I'll leave the redirect in, and resolve this.